data/campaigns/Heir_To_The_Throne/scenarios/06_The_Siege_of_Elensefar.cfg

```diff
@@ -174,20 +157,22 @@
             "Elvish Marshal,Elvish Champion,Elvish Avenger,Elvish Sharpshooter,Elvish Shyde,Elvish Enchantress,Elvish Outrider," +
             "Paladin,Grand Knight," +
             "Mage of Light,Arch Mage,Silver Mage," +
-            "Merman Triton,Merman Hoplite,Merman Javelineer,Merman Entangler,Mermaid Diviner,Mermaid Siren," +
             "Highwayman,Fugitive,Huntsman,Ranger," +
 
             "Elvish Captain,Elvish Hero,Elvish Ranger,Elvish Marksman,Elvish Druid,Elvish Sorceress,Elvish Rider," +
             "Knight,Lancer," +
             "White Mage,Red Mage," +
-            "Merman Warrior,Merman Spearman,Merman Netcaster,Mermaid Priestess,Mermaid Enchantress," +
             "Bandit,Outlaw,Trapper," +
 
             "Elvish Fighter,Elvish Archer,Elvish Shaman,Elvish Scout," +
             "Horseman," +
             "Mage," +
-            "Merman Fighter,Merman Hunter,Mermaid Initiate," +
-            "Thug,Footpad,Poacher"
+            "Thug,Footpad,Poacher" +
+
+            # Merfolk at the bottom because they are not as useful as the other options with this castle placement.
```

Comment:
I agree with you. Apart from S4, S5B, S11, S18 (just one level 3), S19C, S20B...Merfolk don't really have much use in HttT.